pkg/controller/model/grafanaDeployment.go

```diff
@@ -496,8 +496,14 @@ func getContainers(cr *v1alpha1.Grafana, configHash, dsHash string) []v13.Contai
 
 func getInitContainers(cr *v1alpha1.Grafana, plugins string) []v13.Container {
 	cfg := config.GetControllerConfig()
-	image := cfg.GetConfigString(config.ConfigPluginsInitContainerImage, config.PluginsInitContainerImage)
-	tag := cfg.GetConfigString(config.ConfigPluginsInitContainerTag, config.PluginsInitContainerTag)
+	var image string
```

Comment:
```suggestion
     var image string
     var tag string

	if cr.Spec.InitImage != "" {
		image = cr.Spec.InitImage
	} else {
		image = cfg.GetConfigString(config.ConfigPluginsInitContainerImage, config.PluginsInitContainerImage)
		tag = cfg.GetConfigString(config.ConfigPluginsInitContainerTag, config.PluginsInitContainerTag)
	}
```

Reply:
we should assign to an already existing variable, rather than create a new one in the `else{..}`